tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/parser/GradleDependencyUpdater.java

```diff
@@ -37,19 +38,27 @@ public GradleDependencyUpdater( String scriptContents ) throws MultipleCompilati
         nodes = builder.buildFromString( scriptContents );
     }
 
+    public FindDependenciesVisitor insertDependency( GradleDependency gradleDependency ) throws IOException
```

Comment:
User click quick fix then we will check build.gradle file in current project then insert dependence.  Yes. we have junit case test for no dependencies code block. we will add a new dependencies block if there is no dependencies.